domains/rabug.localplayer.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "My portfolio",
+    "domain": "localplayer.dev",
+    "subdomain": "rabug",
+    
+    "owner": {
+        "repo": "https://github.com/NotRabug50/register-localplayer",
+        "email": "rabug50@gmail.com"
+    },
+    
+    "record": {
+        "A": ["76.76.21.21"]
```

Comment:
Send a preview of the site.

Reply:
https://notrabug.vercel.app
![Screenshot_20241011_075037_Brave.jpg](https://github.com/user-attachments/assets/b3f3849e-0ae5-460c-bd02-30815fdf0e64)